src/main/java/net/minecraftforge/fml/common/eventhandler/EventBus.java

```diff
@@ -151,7 +151,7 @@ public void invoke(Event event)
             ArrayList<IEventListener> others = listeners.computeIfAbsent(target, k -> new ArrayList<>());
             others.add(listener);
         }
-        catch (Exception e)
+        catch (Throwable e)
```

Comment:
MethodHandle#invoke, which throws Throwable, is used to construct the lambda class instances. The call to [getHandler](https://github.com/MinecraftForge/MinecraftForge/pull/4118/files#diff-10e669c4a744e9a7418290e95a383028R59) could be surrounded with a try-catch and any non-Exceptions wrapped in an Exception subclass and then thrown?